apps/kyb-app/src/components/organisms/UIRenderer/elements/JSONForm/hocs/withDynamicUIInput/withDynamicUIInput.tsx

```diff
@@ -128,12 +128,16 @@ export const withDynamicUIInput = (
           formData={value}
           definition={definition}
           inputIndex={inputIndex}
+          testId={definition.name}
```

Comment:
I see `definition.name` is used more than once as a `testId`. Are we at risk of an id conflict?

Reply:
No conflicts. Other places that using `definition.name` are not using this HOC. Even though complex components adding suffix to test-id to avoid conficts and test-id works as base.